var/spack/repos/builtin/packages/fortrilinos/package.py

```diff
@@ -24,21 +24,33 @@ class Fortrilinos(CMakePackage):
     """
 
     homepage = "https://trilinos.github.io/ForTrilinos/"
-    url      = "https://github.com/trilinos/ForTrilinos/archive/v2.0.0-dev1.tar.gz"
+    url      = "https://github.com/trilinos/ForTrilinos/archive/v2.0.0.tar.gz"
     git      = "https://github.com/trilinos/ForTrilinos.git"
 
     maintainers = ['sethrj', 'aprokop']
 
-    version('2.0.0-dev2', sha256='2a55c668b3fe986583658d272eab2dc076b291a5f2eb582a02602db86a32030b')
-    version('2.0.0-dev1', sha256='ab664ce2d7fe75c524d7ff6b1efffa3e459ab5739a916e6ea810ae40f39ca4f4')
+    version('2.0.0', sha256='9af3b3eea9934e44d74654a5fa822de08bd0efa43e06e4a4e35a777781f542d6')
+    # Note: spack version comparison implies Version('2.0.0') <
+    # Version('2.0.0-dev1'), so this is the best workaround I could find.
+    version('2.0.dev3',
+            sha256='c20a34b374a56b050bc1db0be1d3db63fca3e59c5803af0cb851b044ac84e6b3',
+            url="https://github.com/trilinos/ForTrilinos/archive/v2.0.0-dev3.tar.gz")
+    version('2.0.dev2',
+            sha256='2a55c668b3fe986583658d272eab2dc076b291a5f2eb582a02602db86a32030b',
+            url="https://github.com/trilinos/ForTrilinos/archive/v2.0.0-dev2.tar.gz")
+    version('2.0.dev1',
+            sha256='ab664ce2d7fe75c524d7ff6b1efffa3e459ab5739a916e6ea810ae40f39ca4f4',
+            url="https://github.com/trilinos/ForTrilinos/archive/v2.0.0-dev1.tar.gz")
```

Comment:
You should probably use `url_for_version()` to do this automatically.

Reply:
I started to and then deleted it -- it would be more than the 3 extra lines that these three exceptional cases take. Do you think it's worth it?

Reply:
If the number of versions proliferates, it's going to reach that point anyway. If you only expect to not have `-dev*` in the future, it may be overkill.

Reply:
Yeah, that was my expectation: the `-dev` versions were IIRC a workaround for how to make semver-friendly versions while keeping the new code at "2.x" while tagging 2.0 as the first complete version. If we ever have to add  more `dev` versions I agree we should push these extra urls into `url_for_version`.

Reply:
Is there a reason to keep older `-dev` now that 2.0.0 has been released?

Reply:
I left it in there for potential users who haven't migrated to trilinos 13, and as a "computer-friendly" way of mirroring the version requirements in the fortrilinos docs. Also spack doesn't have an established procedure for removing older versions :\